packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -52,17 +52,27 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
     },
     apm: {
       kibanaSettings: `${KIBANA_DOCS}apm-settings-in-kibana.html`,
-      supportedServiceMaps: `${KIBANA_DOCS}service-maps.html#service-maps-supported`,
-      customLinks: `${KIBANA_DOCS}custom-links.html`,
+      supportedServiceMaps: isServerless
+        ? `${SERVERLESS_DOCS}apm-service-map`
+        : `${KIBANA_DOCS}service-maps.html#service-maps-supported`,
+      customLinks: isServerless
+        ? `${SERVERLESS_DOCS}apm-create-custom-links`
+        : `${KIBANA_DOCS}custom-links.html`,
       droppedTransactionSpans: `${APM_DOCS}guide/${DOC_LINK_VERSION}/data-model-spans.html#data-model-dropped-spans`,
       upgrading: `${APM_DOCS}guide/${DOC_LINK_VERSION}/upgrade.html`,
       metaData: `${APM_DOCS}guide/${DOC_LINK_VERSION}/data-model-metadata.html`,
       overview: `${APM_DOCS}guide/${DOC_LINK_VERSION}/apm-overview.html`,
-      tailSamplingPolicies: `${APM_DOCS}guide/${DOC_LINK_VERSION}/configure-tail-based-sampling.html`,
+      tailSamplingPolicies: isServerless
+        ? `${SERVERLESS_DOCS}apm-transaction-sampling`
+        : `${APM_DOCS}guide/${DOC_LINK_VERSION}/configure-tail-based-sampling.html`,
       elasticAgent: `${APM_DOCS}guide/${DOC_LINK_VERSION}/upgrade-to-apm-integration.html`,
       storageExplorer: `${KIBANA_DOCS}storage-explorer.html`,
-      spanCompression: `${APM_DOCS}guide/${DOC_LINK_VERSION}/span-compression.html`,
-      transactionSampling: `${APM_DOCS}guide/${DOC_LINK_VERSION}/sampling.html`,
+      spanCompression: isServerless
+        ? `${SERVERLESS_DOCS}apm-compress-spans`
+        : `${APM_DOCS}guide/${DOC_LINK_VERSION}/span-compression.html`,
```

Comment:
This should link to the Observability Guide: https://www.elastic.co/guide/en/observability/current/span-compression.html

Reply:
```suggestion
        : `${OBSERVABILITY_DOCS}/span-compression.html`,
```